CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
+- (Apps) The Connection Terminated dialog will no longer appear in Moonlight when the app on the host exits normally
+- (Systray/Windows) Quitting Sunshine via the system tray will now stop the Sunshine Service rather than leaving it running and allowing it to restart Sunshine
+- (UI) The 100.64.0.0/10 CGN IP address range is now treated as a LAN address range
+- (Video) Removed a workaround for some versions of Moonlight prior to mid-2022
+- (UI) The PIN field is now cleared after successfully pairing
+- (UI) User names are now treated as case-insensitive
+- (Linux) Changed udev rule to automatically grant access to virtual input devices
+- (UI) Several item descriptions were adjusted to reflect newer configuration recommendations
+- (UI) Placeholder text opacity has been reduced to improve contrast with non-placeholder text
+- (Video/Windows) Minor capture performance improvements
+
+**Fixed**
+- (Video) VRAM usage while streaming is significantly reduced, particularly with higher display resolutions and HDR
+- (UPnP) UPnP support was rewritten to fix several major issues handling router restarts, IP address changes, and port forwarding expiration
+- (Input) Fixed modifier keys from the software keyboard on Android clients
+- (Audio) Fixed handling of default audio device changes while streaming
+- (Windows) Fixed streaming after Microsoft Remote Desktop or Fast User Switching has been used
+- (Input) Fixed some games not recognizing emulated Guide button presses
+- (Video/Windows) Fixed incorrect gamma when using an Advanced Color SDR display on the host
+- (Installer/Windows) The installer is no longer blurry on High DPI systems
+- (Systray/Windows) Fixed multiple system tray icons appearing if Sunshine exits unexpectedly
+- (Systray/Windows) Fixed the system tray icon not appearing in several situations
+- (Windows) Fixed hang on shutdown/restart if mDNS registration fails
+- (UI) Fixed missing response headers
+- (Stability) Fixed several possible crashes in cases where the client did not successfully connect
+- (Stability) Fixed several memory leaks
+- (Input/Windows) Back/Select input now correctly generates the Share button when emulating DS4 controllers
+- (Audio/Windows) Fixed various bugs in audio-info.exe that led to inaccurate output on some systems
+- (Video/Windows) Fixed incorrect resolution values from dxgi-info.exe on High DPI systems
+- (Video/Linux) Fixed poor quality encoding from H.264 on Intel encoders
+- (Config) Fixed a couple of typos in predefined resolutions
+
+**Dependencies**
+- Bump sphinx-copybutton from 0.5.1 to 0.5.2
+- Bump sphinx from 6.13 to 7.0.1
+- Bump third-party/nv-codec-headers from 2055784 to 2cd175b
+- Bump furo from 2023.3.27 to 2023.5.20
+
+**Misc**
+- (Build/Linux) Add X11 to PLATFORM_LIBARIES when found
+- (Build/macOS) Fix compilation with Clang 14
+- (Build) Fix header guards on system tray
```

Comment:
I remember this PR, but I think shortly after I removed header guards all around (actually in the `(Docs) Improvements to source code documentation` entry.